### ISSUE TITLE
Add a method to indicate road continues for an unspecified distance

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#8461]
Let's say we are mapping a path, when all the sudden a group of gorillas appears, and we beat a hasty retreat.
Plus it's the end of the semester, so we can't continue mapping tomorrow.
Well instead of
============
which means the road/path is only that long, we would like a way to map
============...
which means (to many people hopefully) that the road continues for an unspecified distance.

**Comment by RobJN** [comment]
Many people add a tag to the last node in the way. Both fixme=continue and fixme=stub are in use.

**Comment by jidanni** [comment]
I'm glad people are making good use of the fixme function. However, as you see I've also filed a rendering issue. It may take years for somebody to get back to that trail. In the meantime there should be something visible on a map to let people know the trail isn't necessarily a dead end in the first place.

**Comment by jidanni** [comment]
So why don't commercial maps have something like this? Well probably because their mappers' paycheck ends at 5 p.m. so it's time to go home and who cares.

Or, they are based on government maps, where the mapper does not have have orders from his superiors to do something like this.

**Comment by jidanni** [comment]
Yes maybe such a new tag that crosses both space and time needs a formal proposal and a referendum and a discussion and everything. Well that's all over my head so I'll just leave that in your hands!

**Comment by jidanni** [comment]
Yes, "dead end signs" could be used to indicate the opposite case. But that would crowd the map with e.g., 995 of them, instead of just five of my "..." ellipsis.

**Comment by jidanni** [comment]
It seems not only for lines, but for areas, a  similar feature is needed. For instance we know a patch of quicksand, or a swamp, starts here, but we really aren't very sure where it ends. So we want to indicate that on the map.

We don't want to just fake it with an arbitrarily large area. And we also don't want to leave invisible fixmes, as we are trying to communicate with the public, and not just fellow mappers.

**Comment by RobJN** [comment]
> I'm glad people are making good use of the fixme function. However, as you see I've also filed a rendering issue. It may take years for somebody to get back to that trail. In the meantime there should be something visible on the map to let people know the trail isn't necessarily a dead end in the first place.

Oh I see. So the ticket here is for changing the rendering style in iD? At first I thought you were after a way to add tags. Perhaps it's worth changing the title to clarify the ask here.

**Comment by jidanni** [comment]
Well I was dreaming of "full support of my new concept", "with additions to the editor, the renderer, and the wiki". But alas ...

**Comment by sommerluk** [comment]
Duplicate of #4375

**Comment by k-yle** [comment]
See https://github.com/openstreetmap/id-tagging-schema/issues/558
